scripts/main/contextMenu.js

```diff
@@ -416,6 +416,7 @@ contextMenu.photoMore = function (photoID, e) {
 	let items = [
 		{ title: build.iconic("fullscreen-enter") + lychee.locale["FULL_PHOTO"], visible: !!showFull, fn: () => window.open(photo.getDirectLink()) },
 		{ title: build.iconic("cloud-download") + lychee.locale["DOWNLOAD"], visible: !!showDownload, fn: () => photo.getArchive([photoID]) },
+		{ title: build.iconic("cloud-download") + lychee.locale["QR_CODE"], fn: () => photo.qrCode([photoID]) },
```

Comment:
`QR_CODE` string needs to be added in:
https://github.com/LycheeOrg/Lychee-front/blob/master/scripts/main/lychee_locale.js
and in 
https://github.com/LycheeOrg/Lychee/tree/master/app/Locale

Reply:
https://github.com/LycheeOrg/Lychee/pull/1260 and https://github.com/LycheeOrg/Lychee-front/pull/287/commits/b47a95be596ef241b72c8bf96ea6b289912204a9